src/device/rdk/applications/launch_with_content.rs

```diff
@@ -132,84 +132,100 @@ pub fn process(_packet: String) -> Result<String, String> {
             app_created = true;
         }
     }
-
-    if app_created {
-        // ****************** Youtube.1.deeplink ********************
-        #[derive(Serialize)]
-        struct Param {
-            url: String,
-        }
-        #[derive(Serialize)]
-        struct RdkRequest {
-            jsonrpc: String,
-            id: i32,
-            method: String,
-            params: Param,
+    let is_cobalt = Dab_Request.appId == "Cobalt" || Dab_Request.appId == "Youtube";
+    let mut param_list = vec![];
+    if (is_cobalt) {
+        if !(Dab_Request.contentId.is_empty()) {
+            param_list.push(format!("v={}",Dab_Request.contentId.clone()));
         }
+    }
 
-        let req_params = Param {
-            url: Dab_Request.contentId,
-        };
-        let request = RdkRequest {
-            jsonrpc: "2.0".into(),
-            id: 3,
-            method: Dab_Request.appId.clone() + ".1.deeplink".into(),
-            params: req_params,
-        };
-        let json_string = serde_json::to_string(&request).unwrap();
-        let response_json = http_post(json_string);
 
-        match response_json {
-            Err(err) => {
-                println!("Erro: {}", err);
+    if (Dab_Request.parameters.len() > 0) {
```

Comment:
remove parenthesis to avoid warning "unnecessary parentheses around if condition"